src/booleans.js

```diff
@@ -1,61 +1,86 @@
+/* eslint-disable no-unreachable */
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !(a || b);
 };
 
 function one(a, b) {
-  // your code here
+  if ((a === true && b === false) || (a === false && b === true)) {
+    return true;
+  } else return false;
 };
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return a > b;
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return a <= b
 };
 
 function isOdd(a) {
-  // your code here
+  if (a%2 !== 0) {
+  return true;
+  } else return false;
```

Comment:
This could be simplified:
```suggestion
  return a % 2 !== 0;
```